main.go

```diff
@@ -117,6 +117,7 @@ import (
 	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/internal_redirect/previous_routes/v3"
 	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/internal_redirect/safe_cross_scheme/v3"
 	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/load_balancing_policies/least_request/v3"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/load_balancing_policies/pick_first/v3"
```

Comment:
Is there any kind of package in the root of go-control-plane somewhere we can import instead?  This list is huge.  :)

Or maybe we can generate the import list instead of maintaining it by hand?

Reply:
Thanks for the review!

1. I can't find a common package that export all the stuff. I suspect even if that exists, it might just be a manual list maintained by go-control-plane so it really does not solve the issue.
2. As for auto-generate, I am not sure what is the criteria for filtering. If I just filter by "*.pb.go" under `go-control-plane/envoy/` there will be O(1000) of them; even `go-control-plane/envoy/extensions` gives me ~O(400).  I am not sure if we need all of them.

I will open a separate issue to track this effort.

Reply:
What is the criteria for including in this binary?

Anyway, PR LGTM; I will merge it.